Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -177,29 +176,7 @@ class ContentBlockerHelper {
         if let enabled = isUserEnabled {
             return enabled
         }
-
-        guard let tab = tab else {
-            return false
-        }
-
-        switch tab.type {
-        case .regular:
-            return isEnabledInNormalBrowsing
-        case .private:
-            return isEnabledInPrivateBrowsing
-        }
-    }
-
-    var status: BlockerStatus {
-        guard isEnabled else {
-            return .Disabled
-        }
-        if stats.total == 0 {
-            // TODO: 161, _may_ need to handle whitelisted situations here
-            return .NoBlockedURLs
-        } else {
-            return .Blocking
-        }
+        return tab != nil
```

Comment:
Maybe just:

`return isUserEnabled == true || tab != nil`

Reply:
Unfortunately this isn't the same logic then. If `isUserEnabled` is non-nil it can still return its value (even if its false)

Reply:
Ah yes, how about:

`return isUserEnabled ?? tab != nil`

?!